### PR TITLE
AT_015.001 | Header > Support > Ask a question > Not checking eCAPTCHA checkbox

### PR DESCRIPTION
**Pull request by TimothyHuxx**
https://trello.com/c/Ik1OQrCz/403-at015001-header-support-ask-a-question-not-checking-ecaptcha-checkbox

### CODE REVIEW

**Review by LIliia149 — APPROVED** [review]
Approved

**Review by Alzhan-B — APPROVED** [review]
Looks good.

File: cypress/e2e/jScript_group.cy.js [hunk 1]
```diff
@@ -215,5 +215,20 @@ describe('Group jScript_group', () => {
         cy.get('a.round[href*="blog"]').invoke('removeAttr', 'target').click();
         cy.url().should('include', '/blog/category/weather');
     })
+
+    it('AT_015.001 | Header > Support > Ask a question > Not checking eCAPTCHA checkbox', () => {
+        cy.visit('https://openweathermap.org/');
+        cy.get('#support-dropdown').click();
+        cy.get('#support-dropdown+ul > li:nth-child(3) > a').invoke('removeAttr', 'target').click();
+        cy.get('.headline').should('have.text', 'Ask a question');
+
```
Comment (shabaiev) [inline cypress/e2e/jScript_group.cy.js:224]: @TimothyHuxx is there any particular reason why you need spaces on lines 224 and 230?

File: cypress/e2e/jScript_group.cy.js [hunk 2]
```diff
@@ -215,5 +215,20 @@ describe('Group jScript_group', () => {
...
+        cy.get('#question_form_is_user_false').check();
+        cy.get('#question_form_email').type('testemail@gmail.com');
```
Comment (shabaiev) [inline cypress/e2e/jScript_group.cy.js:226]: Ideally, this needs to be placed into the 'fixture file'

File: cypress/e2e/jScript_group.cy.js (hunk 1 above)
Comment (TimothyHuxx) [reply cypress/e2e/jScript_group.cy.js:224]: @shabaiev Yes. Maria said to divide our tests into three parts Precondition, Steps< and Expected result.

Comment (shabaiev) [reply cypress/e2e/jScript_group.cy.js:224]: @TimothyHuxx gotcha

File: cypress/e2e/jScript_group.cy.js (hunk 2 above)
Comment (TimothyHuxx) [reply cypress/e2e/jScript_group.cy.js:226]: @shabaiev Thank you for pointing this out. I will create the fixture file for our group next, and we will refactor our tests.

**Review by shabaiev — APPROVED** [review]

**Review by adiuzhikov — DISMISSED** [review]
Approved

**Review by shabaiev — DISMISSED** [review]

**Review by LIliia149 — APPROVED** [review]
approved

**Review by adiuzhikov — APPROVED** [review]
Approved

**Pull request merged**